### ISSUE TITLE
pas de mode bridge

### ISSUE THREAD

**Issue by kgersen** [LaFibre-info/Freepro#6]
Ceci permettrait de palier aux nombreuses limitations.

**Comment by grazzinigerard** [comment]
me demande si un tag supplementaire a celui de bug aurrais du sens ? en tous les cas, perso me semble que cette possibilité permettrais de contourner d'autres problemes ... a suivre

**Comment by kgersen** [comment]
oui "bug" c'est large dans le contexte de ce tracker. faut le comprendre comme 'fonctionnalité qui devrait être la et qui ne l'est pas".

**Comment by jpmat296** [comment]
Mais le potentiel mode bridge désactivera le backup 4G non ?

**Comment by kgersen** [comment]
> Mais le potentiel mode bridge désactivera le backup 4G non ?

pas forcement non.
et vu le niveau technique du backup4G ce n'est pas une grande perte... un simple routeur double-wan fera pareil et celui qui met son routeur derriere une freebox en bridge sait ce qu'il fait...

**Comment by Daleksek** [comment]
https://twitter.com/KevinPolizzi/status/1421199422994661379

**Comment by HectordeMars** [comment]
Bonjour, 

Si les entreprises veulent absolument un mode bridge, il faut qu'elle utilise le port 10G sur la partie haute de de Freebox Pro, couplé à un switch, avec une passerelle définit sur un masque 255.255.255.123 ;)

**Comment by fxguidet** [comment]
Apparemment c'est pour janvier le bridge !

**Comment by Daleksek** [comment]
> 
> 
> Apparemment c'est pour janvier le bridge !

Ou as tu eu cette info ?

**Comment by fxguidet** [comment]
Du tech support que j'ai eu en ligne en fin d'apres midi

**Comment by niclav44** [comment]
Quelqu'un a des nouvelles du mode bridge ?

**Comment by Daleksek** [comment]
Vu le peu de nouveautés sur les MAJ du firmware, je ne pense pas que ça soit disponible de sitôt.

**Comment by johnqube** [comment]
une offre avec un ONT bridgé annoncée cet été (source chef des ventes canal revendeurs qui a bien pris cher avec cette question posée 1000 fois :-)

**Comment by Daleksek** [comment]
> une offre avec un ONT bridgé annoncée cet été (source chef des ventes canal revendeurs qui a bien pris cher avec cette question posée 1000 fois :-)

En espèrent qu'ils ne le limitent pas aux revendeurs, ça apporterait une vraie plus value par rapport aux concurrents .

Ils ont toujours pas améliorer le routage de mon côté, avec une latence toujours élevée après 1 an, je songe à partir

**Comment by mlgd** [comment]
Est ce que ce sujet avance encore ?

Après  an d'abonnement, pas ou peu d'évolution.
Comme @Daleksek, aucune amélioration de la latence, dans un même bureau où Orange Pro me donne 9ms, Free Pro est encore à 18ms.
Je croyais en cette révolution qu'annonçait Free Pro, mais la Freebox Delta faisait mieux.

**Comment by gthgame** [comment]
A quand le mode Bridge. ?

**Comment by kgersen** [comment]
Je ne crois pas que la clientèle type de l'offre pro comprenne le mot "bridge" ;) 
Je ne pense pas que Freepro gagne des clients et des parts de marché en ajoutant cette fonctionnalité...
Leur ambition est minimale pour ne pas impacter les offres en direct et bien plus coûteuses de Jaguar Network.

**Comment by Daleksek** [comment]
Exactement, et le fait que le service client ne soit pas joignable par téléphone dans l'offre de base, montre une envie de proposer une offre simple (simpliste), sans trop se prendre la tête.
C'est pour cela que je suis parti de chez eux, ça ne ressemblera pas au Free GP d'antant "pour les geeks", même Free GP s'éloigne de cette voie (ce qui est normal, l'informatique, n'est plus une activité de niche)

**Comment by palybok** [comment]
Free Pro ne prévoit pas d'implémenter ces fonctions.
https://lafibre.info/free-pro/freebox-pro-analyse-entre-lonu-et-la-box

**Comment by fluorfr** [comment]
Hello, 
Toujours rien ? ça commence à faire long

**Comment by Daleksek** [comment]
> Hello, Toujours rien ? ça commence à faire long

Quand on voit le changelog de la dernière version : https://lafibre.info/free-pro/programme-early-access/msg995438/#msg995438

7 mois entre l'avant dernière MAJ et celle-ci, il n'y a vraiment qu'une infime évolution,

Il ne faut pas trop espérer, ou être extrêmement patient
